ui/src/app/components/toast-banner.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import {FlexColumn, FlexRow} from './flex';
+import {ClrIcon} from './icons';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {reactStyles} from 'app/utils';
+import {CSSProperties} from 'react';
+
+const styles = reactStyles({
+  banner: {
+    backgroundColor: colorWithWhiteness(colors.accent, .9),
+    color: colors.primary,
+    fontSize: '12px',
+    marginRight: '1rem',
+    marginTop: '1rem',
+    padding: '0.5rem',
+    width: '300px',
+    borderRadius: '0.5em',
+    position: 'absolute',
+    top: '0',
+    right: '0',
+    zIndex: 103,
+  },
+  title: {
+    fontWeight: 600,
+    fontSize: '14px',
+    width: '80%',
+    lineHeight: '18px',
+  },
+  message: {
+    lineHeight: '20px',
+    marginTop: '.3rem',
+    paddingRight: '.2rem',
+    fontSize: '14px',
+  },
+  footer: {
+    marginTop: '.5rem'
+  },
+  closeIcon: {
+    position: 'absolute',
+    top: '.3rem',
+    right: '.3rem',
+    colors: colors.accent
+  },
+  warningIcon: {
+    color: colors.warning,
+    flex: '0 0 auto',
+    marginRight: '0.4rem',
+  },
+});
+
+const warningBannerStyle = {
+  ...styles.banner,
+  backgroundColor: colorWithWhiteness(colors.highlight, .5),
+};
+
+const warningIcon = <ClrIcon
+  shape={'warning-standard'}
+  class={'is-solid'}
+  size={26}
+  style={styles.warningIcon}
+/>;
+
+interface ToastProps {
+  title: string;
+  message: string;
+  onClose: Function;
+  warningStyle: boolean;
```

Comment:
nit: how about warningBanner rather than warningStyles? Styles sounds more internal to how the class will display the warningBanner, and the class calling should just worry about if the banner is supposed to be a warning one or info

Reply:
now I'm thinking enum...